scipy/special/orthogonal.py

```diff
@@ -2190,6 +2190,9 @@ def sh_legendre(n, monic=False):
 # module directly. (They shouldn't be; it's not in the public API).
 poch = cephes.poch
 
+# eval_chebyu, eval_sh_chebyt and eval_sh_chebyu: These functions are not
+# used in orthogonal.py, they are not in _rootfuns_map, but their names
+# do appear in _evalfuns, so they must be kept.
```

Comment:
@person142 Does this description match your understanding?  Should these functions still be listed in `_evalfuns`?

Reply:
Yeah, that's correct. We've recently cleaned up importing directly from `basic.py`, cleaning up importing directly from `orthogonal.py` might be next. Probably worthy of a very long deprecation cycle though-IIRC it is really done in the wild.